src/views/Rewards/Rewards.js

```diff
@@ -40,17 +40,34 @@ export default function Rewards(props) {
 
   const { infoTokens } = useInfoTokens(library, chainId, active, undefined, undefined);
 
-  // const { data: rewardProof } = useSWR([getTracerServerUrl(chainId, "/user_reward_proof"), account, selectedWeek, chainId], {
-  // fetcher: (url, account, week) => fetch(`${url}&userAddress=${account}&week=${week}`).then((res) => res.json())
-  // });
-
-  const { data: rewardWeeks, error: failedFetchingRewards } = useSWR([getTracerServerUrl(chainId, "/rewards")], {
+  // Fetch all week data from server
+  const { data: allweeksRewardsData, error: failedFetchingRewards } = useSWR([getTracerServerUrl(chainId, "rewards")], {
     fetcher: (...args) => fetch(...args).then((res) => res.json()),
   });
 
+  // Fetch only the latest week's data from server
+  const { data: currentRewardWeek, error: failedFetchingWeekRewards } = useSWR(
+    [getTracerServerUrl(chainId, "rewards", "week=latest")],
+    {
+      fetcher: (...args) => fetch(...args).then((res) => res.json()),
+    }
+  );
```

Comment:
Could just do something like
```suggestion
  const { data: currentRewardWeek, error: failedFetchingWeekRewards } = useSWR(
    [`${getTracerServerUrl(chainId, "rewards")}&week=latest`],
    {
      fetcher: (...args) => fetch(...args).then((res) => res.json()),
    }
  );
```